editions/tw5.com/tiddlers/filters/allafter.tid

```diff
@@ -0,0 +1,17 @@
+caption: allafter
+created: 20151017145021839
+creator: matabele
```

Comment:
Hi @matabele yes, that's right - and do feel free to send further updates if you see any opportunities for improvement.